### PR TITLE
More basefee naming consistency fixes

### PR DESCRIPTION
**Pull request by roberto-bayardo**
**Description**

More baseFee naming cleanup.  Syntactical change only.

### CODE REVIEW

**Review by tynes — APPROVED** [review]

File: op-batcher/flags/flags_test.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ func TestEnvVarFormat(t *testing.T) {
 			})
 			envFlags := envFlagGetter.GetEnvVars()
 			require.True(t, ok, "must be able to cast the flag to an EnvVar interface")
-			require.Equal(t, 1, len(envFlags), "flags should have exactly one env var")
+			require.LessOrEqual(t, 1, len(envFlags), "flags should have at least one env var")
```
Comment (trianglesphere) [inline op-batcher/flags/flags_test.go]: we have another copy of this test in the op-challenger

File: op-service/txmgr/cli.go [hunk 2]
```diff
@@ -139,9 +139,9 @@ func CLIFlagsWithDefaults(envPrefix string, defaults DefaultFlagValues) []cli.Fl
 			EnvVars: prefixEnvVars("TXMGR_FEE_LIMIT_THRESHOLD"),
 		},
 		&cli.Float64Flag{
-			Name:    MinBasefeeFlagName,
-			Usage:   "Enforces a minimum basefee (in GWei) to assume when determining tx fees. Off by default.",
-			EnvVars: prefixEnvVars("TXMGR_MIN_BASEFEE"),
+			Name:    MinBaseFeeFlagName,
+			Usage:   "Enforces a minimum base fee (in GWei) to assume when determining tx fees. Off by default.",
+			EnvVars: append(prefixEnvVars("TXMGR_MIN_BASEFEE"), prefixEnvVars("TXMGR_MIN_BASE_FEE")...),
```
Comment (trianglesphere) [inline op-service/txmgr/cli.go]: I kinda lean to keeping the old one. We typically convert anything that isn't an alphanumeric character to `_` when converting from flag names to env vars, so `--txmgr.min-basefee` -> `TXMGR_MIN_BASEFEE` feels right to me

Comment (roberto-bayardo) [reply op-service/txmgr/cli.go]: ahah; another option then would be to also accept --txmgr.min-base-fee, but I'm fine reverting instead if you prefer.

Comment (roberto-bayardo) [reply op-service/txmgr/cli.go]: reverted.

File: op-batcher/flags/flags_test.go (hunk 1 above)
Comment (roberto-bayardo) [reply op-batcher/flags/flags_test.go]: reverted this change

**Review by trianglesphere — APPROVED** [review]
TYVM

**Pull request merged**